.github/workflows/aws.yml

```diff
@@ -61,68 +54,127 @@ jobs:
-
   test:
     runs-on: ubuntu-18.04
     needs: [setup]
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - database: postgres
+            version: 9-6
+          - database: postgres
+            version: 10
+          - database: postgres
+            version: 11
+          - database: postgres
+            version: 12
+          - database: postgres
+            version: 13          
+
     steps:
       - uses: actions/checkout@v2
 
-      - name: Update Harness Config
+      - name: Setup Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          cli_config_credentials_token: ${{ secrets.TF_API_TOKEN }}
+          terraform_wrapper: false
+          
+      - name: Terraform Init
+        working-directory: src/test/resources/terraform
+        run: terraform init
+
+      - name: Get Public IP
+        id: ip
+        uses: haythem/public-ip@v1.2
+
+      - name: Print Public IP to tfvars file
+        working-directory: src/test/resources/terraform
+        run: |
+          echo 'public_ip = "${{ steps.ip.outputs.ipv4 }}"' >> github.auto.tfvars
+
+      - name: Terraform Apply Security Group Change
+        working-directory: src/test/resources/terraform
+        run: terraform apply --auto-approve
+
+      - name: Get ${{ matrix.database }}-${{ matrix.version }} Database Host URL
+        working-directory: src/test/resources/terraform
+        run: |
+          echo "DBENDPOINT=$(terraform output -json dbEndpoint | jq -r '."${{ matrix.database }}-${{ matrix.version }}"')" >> $GITHUB_ENV
+
+      - uses: liquibase/liquibase-github-action@v3
+        with:
+          operation: "update"
+          classpath: "src/test/resources/terraform/changelogs"
+          changeLogFile: "postgres.sql"
+          username: "lbuser"
+          password: "LbRootPass1"
+          url: "jdbc:postgresql://${{ env.DBENDPOINT }}/lbcat"
+
+      - name: Update ${{ matrix.database }}-${{ matrix.version }} Harness Config
         working-directory: src/test/resources
-        run: sed -i "s/DBHOST/$DBHOST/g" harness-config.yml
+        run: |
+          sed -i "s/DBENDPOINT/${{ env.DBENDPOINT }}/g" harness-config.yml
+          sed -i "s/DBVERSION/${{ matrix.version }}/g" harness-config.yml
 
-      - name: AWS RDS Postgres Test Run
-        run: mvn -Dtest=LiquibaseHarnessSuiteTest -DdbName=aws-postgresql -Dmaven.test.failure.ignore=true test
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+          restore-keys: |
+            ${{ runner.os }}-maven-
+
+      - name: AWS RDS ${{ matrix.database }}-${{ matrix.version }} Test Run
+        run: mvn -Dtest=LiquibaseHarnessSuiteTest -DdbName=aws-postgresql -DdbVersion=${{ matrix.version }} -Dmaven.test.failure.ignore=true test
```

Comment:
This could be `aws-${{ matrix.database }}` and allow more reuse-ability later.

Reply:
fixed in the oracle branch (DAT-8039)